spec/services/frontend_logger_spec.rb

```diff
@@ -7,11 +7,11 @@ def example_method_handler(ok:, **rest)
     end
   end
 
-  class ExampleAnalytics < FakeAnalytics
+  class ExampleAnalyticsForFL < FakeAnalytics
     include ExampleAnalyticsEvents
   end
```

Comment:
ooo I have thoughts about this, I like to create anonymous classes so we don't have to worry about these collisions:

```ruby
let(:analytics_class) do
  Class.new(FakeAnalytics) do
    include ExampleAnalyticsEvents
  end
end
let(:analytics) { analytics_class.new }
```

Reply:
TIL! Added: 1715e9de4c5e0577bac4cc37464c21dcd133c254